src/main/java/com/autotune/database/service/ExperimentDBService.java

```diff
@@ -94,27 +95,38 @@ public boolean getRecommendationToSave(Map<String, KruizeObject> experimentsMap,
      * The method then converts the retrieved data into KruizeObject format, adds them to a list,
      * and sends it to the ExperimentInterface implementation to store the objects.
      */
-    public void loadAllExperiments() throws Exception {
+    public void loadAllExperimentsData() throws Exception {
         List<KruizeExperimentEntry> entries = experimentDAO.loadAllExperiments();
         List<CreateExperimentAPIObject> createExperimentAPIObjects = DBHelpers.Converters.KruizeObjectConverters.convertExperimentEntryToCreateExperimentAPIObject(entries);
         List<KruizeObject> kruizeExpList = new ArrayList<>();
+        List<KruizeResultsEntry> kruizeResultsEntries = experimentDAO.loadAllResults();
+        List<UpdateResultsAPIObject> updateResultsAPIObjects = DBHelpers.Converters.KruizeObjectConverters.convertResultEntryToUpdateResultsAPIObject(kruizeResultsEntries);
+        List<ExperimentResultData> resultDataList = new ArrayList<>();
+        ExperimentInterface experimentInterface = new ExperimentInterfaceImpl();
+
         int failureThreshHold = createExperimentAPIObjects.size();
         int failureCount = 0;
         for (CreateExperimentAPIObject createExperimentAPIObject : createExperimentAPIObjects) {
             KruizeObject kruizeObject = Converters.KruizeObjectConverters.convertCreateExperimentAPIObjToKruizeObject(createExperimentAPIObject);
-            if (null != kruizeObject)
+            if (null != kruizeObject) {
                 kruizeExpList.add(kruizeObject);
+            }
             else {
                 failureCount++;
             }
         }
         if (failureThreshHold > 0 && failureCount == failureThreshHold) {
             throw new Exception("None of the experiments are able to load from DB.");
         }
-        ExperimentInterface experimentInterface = new ExperimentInterfaceImpl();
         experimentInterface.addExperimentToLocalStorage(KruizeOperator.autotuneObjectMap, kruizeExpList);
-        //TODO get KruizeResultsEntry to KruizeObject.kubernetes_objects.containerDataMap.results -> Saad
-        //TODO get KruizeRecommendationEntry to KruizeObject.kubernetes_objects.containerDataMap.containerRecommendations -> Bharath
+
```

Comment:
All update results code should start from here

Reply:
Moved it now